### PR TITLE
feat(SlimeRead): add presence

### PR DESCRIPTION
**Pull request by Luscarvalho**
## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Some images </summary>

home

image

manga page

image

reader page

image

</details>

### CODE REVIEW

File: websites/S/SlimeRead/metadata.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+	"description": {
+		"en": "Discover and enjoy a vast collection of literary works, including manga, manhwa, manhua, novels, and much more.",
+		"pt_BR": "Descubra e desfrute de uma vasta coleção de obras literárias, incluindo mangás, manwhas, manhuas, novels e muito mais."
+	},
+	"url": [
+		"slimeread.com",
+		"www.slimeread.com"
+	],
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/Ht1LSYT.png",
```
Comment (DarkVillager) [inline websites/S/SlimeRead/metadata.json:17]: This is without the background:
```suggestion
	"logo": "https://i.imgur.com/x6NuCdk.png",
```

File: websites/S/SlimeRead/presence.ts [hunk 2]
```diff
@@ -0,0 +1,91 @@
+const presence = new Presence({
+		clientId: "1190448612778463242",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/Ht1LSYT.png",
```
Comment (DarkVillager) [inline websites/S/SlimeRead/presence.ts:8]: This is without the background:
```suggestion
			largeImageKey: "https://i.imgur.com/x6NuCdk.png",
```

File: websites/S/SlimeRead/presence.ts [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+				"p.text-dark.dark\\:text-light.transition.duration-300.text-2xl.font-bold.font-serif"
+			)?.textContent;
+			presenceData.buttons = [{ label: "Ler mangá", url: href }];
+			break;
+		}
+
+		case "ler": {
+			const chapterInfo = document.querySelector(
+					"h1.font-serif.font-bold.text-xl.text-center.sm\\:text-left.text-dark.dark\\:text-light.transition.duration-300"
+				)?.textContent,
```
Comment (DarkVillager) [inline websites/S/SlimeRead/presence.ts:67]:
```suggestion
			const chapterInfo = document.querySelector(
					'[class="ml-4 mt-4"] > h1'
				)?.textContent,
```

File: websites/S/SlimeRead/presence.ts [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+				mediaLink = document.querySelector<HTMLAnchorElement>(
+					"a.hover\\:text-primary-700.dark\\:hover\\:text-primary-500.hover\\:underline.transition.duration-300.link.cursor-pointer"
+				).href,
```
Comment (DarkVillager) [inline websites/S/SlimeRead/presence.ts:70]:
```suggestion
				mediaLink = document.querySelector<HTMLAnchorElement>(
					'[class="ml-4 mt-4"] > h1 > a'
				)?.href,
```

File: websites/S/SlimeRead/presence.ts [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+				if (searchValue) presenceData.state = `Mangá: ${searchValue}`;
+			}
+			break;
+
+		case "manga": {
+			presenceData.largeImageKey = document.querySelectorAll<HTMLImageElement>(
+				'img[title="Imagem"]'
+			)[2].src;
+			presenceData.details = document.querySelector(
+				"p.text-dark.dark\\:text-light.transition.duration-300.text-2xl.font-bold.font-serif"
```
Comment (DarkVillager) [inline websites/S/SlimeRead/presence.ts:58]: Also try to shorten selectors here

**Review by DarkVillager — CHANGES_REQUESTED** [review]
Try to shorten selectors where possible.

**Review by DarkVillager — APPROVED** [review]
LGTM

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/S/SlimeRead/presence.ts [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+				if (searchValue) presenceData.state = `Mangá: ${searchValue}`;
+			}
+			break;
+
+		case "manga": {
+			presenceData.largeImageKey = document.querySelectorAll<HTMLImageElement>(
+				'img[title="Imagem"]'
+			)[2].src;
+			presenceData.details = document.querySelector(
+				'[class="mt-4 sm:ml-4 sm:mt-0  "] > p'
```
[the remark this answers is not in the document]
Comment (Luscarvalho) [reply websites/S/SlimeRead/presence.ts:58]: Yes

**Review by Bas950 — APPROVED** [review]

**Review by theusaf — APPROVED** [review]

**Pull request merged**